molecule.yml

```diff
@@ -0,0 +1,16 @@
+---
+ansible:
+  config_file: ansible.cfg
+  sudo: True
```

Comment:
Fixed in [`3f90cab7`](https://github.com/azavea/ansible-terraform/pull/1/commits/3f90cab7368c31f9a6eaf19ed2f133759432b4bd)